packages/next/src/server/dev/turbopack-utils.ts

```diff
@@ -460,7 +586,10 @@ export async function handleEntrypoints({
   nextConfig: NextConfigComplete
   rewrites: SetupOpts['fsChecker']['rewrites']
 
+  assetMapper?: AssetMapper // dev
   changeSubscriptions?: ChangeSubscriptions // dev
+  clients?: Set<ws> // dev
+  clientStates?: ClientStateMap // dev
   serverFields?: ServerFields // dev
 
   hooks?: HandleEntrypointsHooks //dev
```

Comment:
maybe this should be in a single `dev` sub object. Could be a separate PR.

```suggestion
  dev?: {
    assetMapper: AssetMapper
    changeSubscriptions: ChangeSubscriptions
    clients: Set<ws>
    clientStates: ClientStateMap
    serverFields: ServerFields

    hooks: HandleEntrypointsHooks
  }
```

Reply:
done, missed the separate PR thing 🙃